README.md

```diff
@@ -390,6 +390,40 @@ Below is a list of currently supported actions. Requests with invalid actions or
     ```
     true
     ```
+*   **guiDeckOverview**
+
+    Opens the Deck Overview screen for the deck with the given name; returns `true` if succeeded or `False` otherwise.
+
+    *Sample request*:
+    ```
+    {
+        "action": "guiDeckOverview",
+		"params": {
+			"name": "Default"
+		}
+    }
+    ```
+
+    *Sample response*:
+    ```
+    true
+    ```
+
+*   **guiDeckBrowser**
+
+    Opens the Deck Browser screen; returns `true` if succeeded or `null` otherwise.
```

Comment:
Please make the API return `true` and `false` instead of `true` and `null` for consistency.